### PR TITLE
New TC [PolicyTableUpdate] "timeout" countdown start

### PR DESCRIPTION
**Pull request by istoimenova**
To run the script you need to replace
user_modules/shared_testcases/testCasesForPolicyTable.lua
user_modules/shared_testcases/testCasesForPolicyTableSnapshot.lua
user_modules/shared_testcases/testCasesForBuildingSDLPolicyFlag.lua
from #32. Common functions are under review.

@maria-mih please review the script according to template.

### CODE REVIEW

File: test_scripts/Polices/Policy_Table_Update/021_ATF_P_TC_timeout_countdown_start.lua [hunk 1]
```diff
@@ -0,0 +1,203 @@
+---------------------------------------------------------------------------------------------
+-- Requirements summary:
+-- [PolicyTableUpdate] PoliciesManager changes status to “UPDATING”
```
Comment (maria-mih) [inline test_scripts/Polices/Policy_Table_Update/021_ATF_P_TC_timeout_countdown_start.lua]: Please change to [PolicyTableUpdate] "timeout" countdown start

File: test_scripts/Polices/Policy_Table_Update/021_ATF_P_TC_timeout_countdown_start.lua [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+-- [HMI API] SystemRequest request/response
+--
+-- Description:
```
Comment (maria-mih) [inline test_scripts/Polices/Policy_Table_Update/021_ATF_P_TC_timeout_countdown_start.lua]: Please check description it should be more like: 
`PoliciesManager must start timeout taken from "timeout_after_x_seconds" field of LocalPT right after OnSystemRequest is sent out to mobile app`.

File: test_scripts/Polices/Policy_Table_Update/021_ATF_P_TC_timeout_countdown_start.lua [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+-- 1. Used preconditions
+-- SDL is built with "-DEXTENDED_POLICY: EXTERNAL_PROPRIETARY" flag
+-- Application is registered.
+-- PTU is requested.
+-- SDL->HMI: SDL.OnStatusUpdate(UPDATE_NEEDED)
+-- SDL->HMI:SDL.PolicyUpdate(file, timeout, retry[])
+-- HMI -> SDL: SDL.GetURLs (<service>)
+-- HMI->SDL: BasicCommunication.OnSystemRequest ('url', requestType:PROPRIETARY, appID="default")
+-- SDL->app: OnSystemRequest ('url', requestType:PROPRIETARY, fileType="JSON", appID)
+-- 2. Performed steps
```
Comment (maria-mih) [inline test_scripts/Polices/Policy_Table_Update/021_ATF_P_TC_timeout_countdown_start.lua]: Please add steps

File: test_scripts/Polices/Policy_Table_Update/021_ATF_P_TC_timeout_countdown_start.lua [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+--
+-- Expected result:
+-- SDL->HMI: SDL.OnStatusUpdate(UPDATING) right after SDL->app: OnSystemRequest
+---------------------------------------------------------------------------------------------
```
Comment (maria-mih) [inline test_scripts/Polices/Policy_Table_Update/021_ATF_P_TC_timeout_countdown_start.lua]: Maybe need to add to expected result: 
`SDL waits for SystemRequest response from <app ID> within 'timeout' value, if no obtained, it starts retry sequence`

**Comment by maria-mih** [comment]
@istoimenova 
Please check comments and proceed

**Comment by istoimenova** [comment]
@LuxoftAKutsan please merge.

**Pull request merged**